starboard/loader_app/installation_manager.cc

```diff
@@ -14,6 +14,7 @@
 
 #include "starboard/loader_app/installation_manager.h"
 
+#include <sys/stat.h>
```

Comment:
c headers should be in a different section than c++ headers, just add a newline after this line